L1Trigger/Phase2L1ParticleFlow/test/make_l1ct_binaryFiles_cfg.py

```diff
@@ -0,0 +1,139 @@
+if not args.patternFilesOFF:
+    process.l1tLayer1Barrel.patternWriters = cms.untracked.VPSet(*barrelWriterConfigs)
+    # process.l1tLayer1Barrel9.patternWriters = cms.untracked.VPSet(*barrel9WriterConfigs) # not enabled for now
+    process.l1tLayer1HGCal.patternWriters = cms.untracked.VPSet(*hgcalWriterConfigs)
+    process.l1tLayer1HGCalNoTK.patternWriters = cms.untracked.VPSet(*hgcalNoTKWriterConfigs)
+    process.l1tLayer1HF.patternWriters = cms.untracked.VPSet(*hfWriterConfigs)
+
+process.runPF = cms.Path( 
+        process.l1tSAMuonsGmt +
+        process.l1tGTTInputProducer +
+        process.l1tVertexFinderEmulator +
+        process.l1tLayer1Barrel +
+        process.l1tLayer1Barrel9 +
+        process.l1tLayer1HGCal +
+        process.l1tLayer1HGCalNoTK +
+        process.l1tLayer1HF +
+        process.l1tLayer1 +
+        process.l1tLayer2Deregionizer +
+        process.l1tLayer2SeedConeJetsCorrected +
+        # process.l1tLayer2SeedConeJetWriter +
+        process.l1tLayer2EG
+    )
+process.runPF.associate(process.L1TLayer1TaskInputsTask)
+
+
+#####################################################################################################################
+## Layer 2 e/gamma 
+
+if not args.patternFilesOFF:
+    process.l1tLayer2EG.writeInPattern = True
+    process.l1tLayer2EG.writeOutPattern = True
+    process.l1tLayer2EG.inPatternFile.maxLinesPerFile = eventsPerFile_*54
+    process.l1tLayer2EG.outPatternFile.maxLinesPerFile = eventsPerFile_*54
+
+#####################################################################################################################
+## Layer 2 seeded-cone jets 
+if not args.patternFilesOFF:
+    process.runPF.insert(process.runPF.index(process.l1tLayer2SeedConeJetsCorrected)+1, process.l1tLayer2SeedConeJetWriter)
+    process.l1tLayer2SeedConeJetWriter.maxLinesPerFile = eventsPerFile_*54
+
+if not args.dumpFilesOFF:
+  for det in "Barrel", "Barrel9", "HGCal", "HGCalNoTK", "HF":
+        l1pf = getattr(process, 'l1tLayer1'+det)
+        l1pf.dumpFileName = cms.untracked.string("TTbar_PU200_"+det+".dump")
+
+
+process.source.fileNames  = [ '/store/cmst3/group/l1tr/gpetrucc/11_1_0/NewInputs110X/110121.done/TTbar_PU200/inputs110X_%d.root' % i for i in (1,3,7,8,9) ]
+process.l1tPFClustersFromL1EGClusters.src = cms.InputTag("L1EGammaClusterEmuProducer",)
+process.l1tPFClustersFromCombinedCaloHCal.phase2barrelCaloTowers = [cms.InputTag("L1EGammaClusterEmuProducer",)]
+process.l1tPFClustersFromHGC3DClusters.src  = cms.InputTag("hgcalBackEndLayer2Producer","HGCalBackendLayer2Processor3DClustering")
+process.l1tPFClustersFromCombinedCaloHF.hcalCandidates = [ cms.InputTag("hgcalBackEndLayer2Producer","HGCalBackendLayer2Processor3DClustering")]
+process.l1tPFTracksFromL1Tracks.L1TrackTag = cms.InputTag("TTTracksFromTrackletEmulation","Level1TTTracks")
+process.l1tGTTInputProducer.l1TracksInputTag = cms.InputTag("TTTracksFromTrackletEmulation","Level1TTTracks")
+
```

Comment:
is this designed for general CMSSW testing/unit testing? Or is this personal use?

Reply:
it's run in the Correlator firmware CI and for development there, but not in standard CMSSW release building & testing